Civi/Test/CiviCodeceptionTrait.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace Civi\Test;
+
+/**
+ * Trait CiviCodeceptionTrait
+ * Trait for common functions used for Codeception End-to-End Testing framework
+ * @package Civi\Test
+ */
+trait CiviCodeceptionTrait {
+
+  /**
+   * Parse Parameters, and create generic Civi URl
+   * @param $page
```

Comment:
The `@param` docbocks should include a little more detail. Example:

```php
  * @ param string $page
  *    The path and parameters of a CiviCRM page. Ex: "civicrm/dashboard?reset=1".
```

The same would apply to other `@param`s.